lib/pg_saurus/connection_adapters/postgresql_adapter/extension_methods.rb

```diff
@@ -52,6 +52,20 @@ def create_extension(extension_name, options = {})
     execute(sql)
   end
 
+  # Execute SQL to load a postgresql extension module into the current database
+  # if it does not already exist. Then reload the type map.
+  #
+  # @param [#to_s] extension_name Name of the extension module to load
+  # @param [Hash] options
+  # @option options [#to_s,nil] :schema_name The name of the schema in which to install the extension's objects
+  # @option options [#to_s,nil] :version The version of the extension to install
+  # @option options [#to_s,nil] :old_version Alternative installation script name
+  #    that absorbs the existing objects into the extension, instead of creating new objects
+  def enable_extension(extension_name, options = {})
+    options[:if_not_exists] = true
+    create_extension(extension_name, options).tap { reload_type_map }
```

Comment:
I'm trying to closely emulate the Rails method: http://apidock.com/rails/ActiveRecord/ConnectionAdapters/PostgreSQLAdapter/enable_extension